app/views/student_mailer/finalizing_notification.html.erb

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <p>Hi <%= @user.name %>,</p>
+    <p>Your group is now in the process of finalizing! Please go to <%= "#{@college.site_url} #{@finalizing_path}" %> to confirm your membership.</p>
```

Comment:
Let's reword this to include language about "confirming the housing group" (see my earlier comment and #352)